flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+	"only-arches": ["x86_64"]
```

Comment:
it's still deceptive to see this while the description mention "cross-platform"

Reply:
@qdot can answer that?

Reply:
We compile for mobile ARM platforms.

Technically correct is best correct.

Reply:
technically mobile platforms include aarch64-linux. Not just Android or iOS....

Reply:
Are you asking us to remove the "cross-platform" wording from this? If so, please just say so.

Reply:
Marking as resolved since Hubert didn't clarify.

Reply:
not sure what was unambiguous about building on aarch64, but apparently you don't want that.

Reply:
This is a single developer project that has had zero requests for that architecture so far. If you would like to use your toys on that platform, please let us know and we will consider accordingly.